test/unit/messaging/validate-message.test.js

```diff
@@ -1,8 +1,7 @@
-const { VALIDATION } = require('../../../app/constants/errors')
-
 jest.mock('../../../app/messaging/message-schema')
 const mockSchema = require('../../../app/messaging/message-schema')
 
+const { VALIDATION } = require('../../../app/constants/errors')
```

Comment:
Space needed to separate the constant from the mocks?

Reply:
Thought it would be cleaner to keep the mock pair together.  Reduces the risk someone will insert something between them and potentially break the mock

Reply:
Apologies I meant that I think there needs to be a space between { VALIDATION } and { REQUEST_MESSAGE } as one is a constant and one is a mocked value - if that makes sense